src/pages/pollingPage.vue

```diff
@@ -32,9 +47,18 @@ const router = useRouter();
 const createPoll = (() => {
   router.push('/addpoll')
 })
+const updateTitle = ((id) => {
+  router.push(`/updatetitle/${id}`)
```

Comment:
page for an update title? No, it will go to update Poll page, from there user can update anything in the poll @ShivanshSinghnnovative